build/grunt.js

```diff
@@ -23,11 +85,13 @@ module.exports = function(grunt) {
    *                       using an object constructed from `bannerCommonData`
    *                       and the `data` argument.
    */
-  let createLicenseProcessor = (data) => function () {
-    return grunt.template.process(license, {
-      data: _.merge({}, bannerCommonData, data)
-    });
-  };
+  function createLicenseProcessor(data) {
```

Comment:
Yeah, stylistic. I typically feel like named functions are more debuggable, but that may just be the old curmudgeon in me.